lib/super_digit.rb

```diff
@@ -1,15 +1,33 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n), with n being the number of digits in the initial integer.
+# Space Complexity - O(n), because of the recursive stack; size of the stack depends on the number of digits.
 def super_digit(n)
-   
+  sum = 0
+
+  if n < 10
+    return sum + n
```

Comment:
Why not just return `n` since sum is 0.

```suggestion
    return n
```